kits/smc-bias-agent/.env.example

```
@@ -0,0 +1,4 @@
+LAMATIC_API_KEY=lt-a37857b83dd6f993e9d327b9a9f68bba
+LAMATIC_GRAPHQL_URL=https://ayushsorganization263-ayushsproject211.lamatic.dev/graphql
+LAMATIC_PROJECT_ID=3bffc6da-08d8-4f1a-af0c-f654bca169e6
+LAMATIC_WORKFLOW_ID=8ba7821c-251d-4d58-827b-bb5d627a3d57
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)